execution/consensus/clique/snapshot_test.go

```diff
@@ -416,7 +417,7 @@ func TestClique(t *testing.T) {
 				}
 			}
 			// Create the genesis block with the initial set of signers
-			genesis := &types.Genesis{
+			genesis := &genesis.Genesis{
```

Comment:
Not good to have package name same as variable name. I don't see a problem with `Genesis` residing in `types`.

Reply:
I don't want to move `Genesis` to `erigon-lib`  because it depends on `ChainConfigByGenesisHash`, which is not generic enough for `erigon-lib`.

Reply:
What is the actual problem? From what I saw `ChainCOnfigByGenesisHash` is just a switch-case over the genesis hashes in params.

Reply:
Yes, but I don't want Polygon-specific things like `BorMainnetGenesisHash` to make it into `erigon-lib`. Ideally all Polygon-specific things should live in `polygon`.

Reply:
But let try something though...